docs/components/carousel.md

```diff
@@ -6,18 +6,17 @@ A description of the component goes here.
 
 ```html preview
 <div class="carousel-options">
-    <sl-switch name="loop"> Loop </sl-switch>
-    <sl-switch name="show-controls"> Show controls </sl-switch>
-    <sl-switch name="show-pagination"> Show pagination </sl-switch>
+  <sl-switch name="loop"> Loop </sl-switch>
+  <sl-switch name="show-controls"> Show controls </sl-switch>
+  <sl-switch name="show-pagination"> Show pagination </sl-switch>
 </div>
 <sl-divider></sl-divider>
 <sl-carousel heading="Example carousel">
-  <sl-carousel-slide><img src="https://picsum.photos/530/300/?random=1" /></sl-carousel-slide>
-  <sl-carousel-slide><img src="https://picsum.photos/530/300/?random=2" /></sl-carousel-slide>
-  <sl-carousel-slide><img src="https://picsum.photos/530/300/?random=3" /></sl-carousel-slide>
-  <sl-carousel-slide><img src="https://picsum.photos/530/300/?random=4" /></sl-carousel-slide>
-<sl-carousel-slide><img src="https://picsum.photos/530/300/?random=5" /></sl-carousel-slide>
```

Comment:
There may be value in keeping this, but maybe call it `<sl-carousel-item>` to match existing conventions.

Without it, can we still slot in non-images? It's not very reliable to use `::slotted()` due to its low specificity, but a wrapper solves that and can provide a default relatively-positioned container for things like HTML slides.

Reply:
I was playing around to see how it would work without the wrapper, tried with a card component as well and it worked pretty well, but yea, it would be better to have a dedicated component for this